src/services/ui/src/pages/actions/setups/setupIssueRai.ts

```diff
@@ -1,15 +1,16 @@
 import { z } from "zod";
+import {
+  zAdditionalInfo,
+  zAttachmentOptional,
+  zAttachmentRequired,
+} from "@/pages/form/zod";
 
 export const defaultIssueRaiSetup = {
   schema: z.object({
-    additionalInformation: z.string().max(4000),
+    additionalInformation: zAdditionalInfo,
```

Comment:
good reuse opportunity there with additional information. It will also lead to better consistency across the board I bet

Reply:
Yep! Merging submission forms into the same pattern will also help consolidate things.